### PR TITLE
feature: Add global_namespace_import to @Symfony ruleset

### PR DESCRIPTION
**Pull request by chalasr**
As per https://github.com/symfony/symfony/pull/48002#issuecomment-1291933825:
> This is a rule we have since years in the Symfony coding standards.

### CODE REVIEW

File: src/RuleSet/Sets/SymfonySet.php
Comment (keradus) [inline src/RuleSet/Sets/SymfonySet.php:62]: Hey!
would you mind raising a PR for Symfony repo to apply code changes that this config would introduce? I wonder if the community will approve exact this config after seeing the results, or maybe they would like to adjust it one way or another

Comment (chalasr) [reply src/RuleSet/Sets/SymfonySet.php:62]: Hey, do you mean a PR changing the code in ways that triggers this rule to make sure we agree? 
We (the core team) are constantly rejecting the addition of `use` statements for all the elements configured here (classes, functions and constants) so I can tell this is what we want :)

File: doc/ruleSets/Symfony.rst
Comment (keradus) [inline doc/ruleSets/Symfony.rst:40]: would you mind to run https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/dev-tools/doc.php on your PR?

Comment (chalasr) [reply doc/ruleSets/Symfony.rst]: nice thanks! done

**Review by julienfalque — APPROVED** [review]

**Comment by julienfalque** [comment]
Thank you @chalasr.

**Pull request merged**
